Resolves #4740 - Add new default date range

jorgecoutinhobr opened the pull request:
Resolves #4740 

### Description
* Change the default date range for all filters to 60 days prior, 30 days in the future.

### Type of change
* New feature (non-breaking change which adds functionality)

### How Has This Been Tested?
* add test cases for default range in date_range_picker_shared_example

jorgecoutinhobr commented:
I removed the "this_year" method from DateRangeHelper because it is no longer used

cielf commented:
> I removed the "this_year" method from DateRangeHelper because it is no longer used

@jorgecoutinhobr  We still want This Year to be available -- just not the default.

jorgecoutinhobr commented:
> > I removed the "this_year" method from DateRangeHelper because it is no longer used
> 
> @jorgecoutinhobr We still want This Year to be available -- just not the default.

@cielf It is still possible to use the "this year" filter, because it is defined in application.js. I removed the "this year" method from DataRangeHelper, because it was only used to set a default date if the params were nil. But if necessary I can return with this method.

![imagem_2024-10-31_085430334](https://github.com/user-attachments/assets/49e723c2-1a5f-46c1-857a-f694fca14a33)

cielf commented:
@jorgecoutinhobr Thanks for the clarification!

cielf reviewed (commented):
Tested it out, and it looks good to me!
@dorner - Do you have any concerns?

dorner commented:
Nope!
